lib/ringflux/plugin.rb

```diff
@@ -13,12 +13,12 @@ class Ringflux::Plugin < Adhearsion::Plugin
     async       true        , desc: 'Write metrics asynchronously'
   end
 
-  run :ringflux, after: :punchblock do
+  init :ringflux, after: :punchblock do
     InfluxDB::Logging.logger = logger
-    new.start
+    new.configure
   end
 
-  def start
+  def configure
     logger.info "Connecting to InfluxDB #{config.username}@#{config.host} with database #{config.database}"
```

Comment:
Change `Connecting to` to `Configuring` since no connection is made at this point